.vscode/settings.json

```diff
@@ -7,5 +7,7 @@
   ],
   "css.validate": false,
   "scss.validate": false,
-  "stylelint.enable": true
+  "stylelint.enable": true,
+  "typescript.validate.enable": true,
+  "javascript.validate.enable": true,
```

Comment:
Turns on type validation in VSCode. See https://github.com/chanzuckerberg/edu-design-system/pull/514